Recast/Discover/MainSearchViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  MainSearchViewController.swift
+//  Recast
+//
+//  Created by Jack Thompson on 9/25/18.
+//  Copyright © 2018 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+protocol SearchTableViewDelegate: class {
+    func refreshController()
+}
+
+class MainSearchViewController: UIViewController {
+
+    // MARK: - Variables
+    var searchController: UISearchController!
+    var searchResultsTableView: UITableView!
+    var tableViewData: MainSearchDataSourceDelegate!
+    var searchDelayTimer: Timer?
+    var searchText: String = ""
+
+    var discoverContainerView: UIView!
+    var discoverVC: DiscoverViewController!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableViewData = MainSearchDataSourceDelegate()
+        tableViewData.delegate = self
+
+        navigationController?.navigationBar.prefersLargeTitles = false
+        navigationController?.navigationBar.backgroundColor = .black
+        navigationController?.navigationBar.barTintColor = .clear
+        navigationController?.navigationBar.isTranslucent = false
+
+        searchController = UISearchController(searchResultsController: nil)
+        searchController.hidesNavigationBarDuringPresentation = false
+        searchController.isActive = true
+        searchController.dimsBackgroundDuringPresentation = false
+
+        searchController.searchBar.delegate = self
+        searchController.searchBar.isTranslucent = true
+        searchController.searchBar.barTintColor = .black
+        searchController.searchBar.barStyle = .black
+        searchController.searchBar.tintColor = .white
+
+        let searchField = searchController.searchBar.value(forKey: "searchField") as? UITextField
+        searchField?.textColor = .white
+        searchField?.backgroundColor = #colorLiteral(red: 0.09749762056, green: 0.09749762056, blue: 0.09749762056, alpha: 1)
+
+        navigationItem.titleView = searchController?.searchBar
+
+        searchResultsTableView = UITableView(frame: .zero, style: .plain)
+        searchResultsTableView.dataSource = tableViewData
+        searchResultsTableView.delegate = tableViewData
+        //swiftlint:disable:next line_length
+        searchResultsTableView.register(PodcastTableViewCell.self, forCellReuseIdentifier: PodcastTableViewCell.cellReuseIdentifier)
```

Comment:
You could probably decrease this line length by pulling PodcastTableViewCell.cellReuseIdentifier out into a `let = reuseId = ...`

Better than ignoring the convention